server/chat-plugins/poll.ts

```diff
@@ -264,6 +264,7 @@ export const commands: ChatCommands = {
 		create: 'new',
 		createmulti: 'new',
 		htmlcreatemulti: 'new',
+		queue: 'new',
```

Comment:
Hm, it looks like you can't queue a multi-poll or an HTML poll.

I think at this point, a lot of this logic might need to be moved out of this command and into a static method on the `Poll` constructor. You probably want a variable for `cmd.includes('queue')` like we do for `multi` and `html`.

Reply:
Okay. Do you want the logic moved in this PR or just a `cmd.includes` for now?

Reply:
`cmd.includes` is fine for now.

Reply:
Okay.